docs/src/components/ComponentBestPractices.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react'
-import _ from 'lodash'
-import { Box, Flex, Header, Icon, List, Segment, ShorthandValue } from '@stardust-ui/react'
+import * as _ from 'lodash'
+import { Box, Flex, Header, Icon, List, Segment, ShorthandCollection } from '@stardust-ui/react'
 import { Extendable } from 'src/types'
 
 export type ComponentBestPracticesProps = Extendable<{
-  doList?: ShorthandValue<{}>[]
-  dontList?: ShorthandValue<{}>[]
+  doList?: ShorthandCollection<{}>
```

Comment:
Do we really need the empty object here?

Reply:
It should be `ShorthandCollection<ListItemProps>`, fixed 👍